android-libkiwixbuild/app/src/androidTest/java/org/kiwix/kiwixlib/ExampleInstrumentedTest.kt

```diff
@@ -0,0 +1,24 @@
+package org.kiwix.kiwixlib
+
+import androidx.test.platform.app.InstrumentationRegistry
+import androidx.test.ext.junit.runners.AndroidJUnit4
+
+import org.junit.Test
+import org.junit.runner.RunWith
+
+import org.junit.Assert.*
+
+/**
+ * Instrumented test, which will execute on an Android device.
+ *
+ * See [testing documentation](http://d.android.com/tools/testing).
+ */
+@RunWith(AndroidJUnit4::class)
+class ExampleInstrumentedTest {
```

Comment:
I'm pretty sure we don't need ExamplnstrumetedTest.kt

Reply:
done

Reply:
done